### PR TITLE
Fix inconsistent text styling

### PR DESCRIPTION
**Pull request by bjosttveit**
## Related Issue(s)

- https://altinn.slack.com/archives/C02EJ9HKQA3/p1723722789648419?thread_ts=1723617628.368379&cid=C02EJ9HKQA3

## Verification/QA

- Manual functionality testing
  - [x] I have tested these changes manually
  - [ ] Creator of the original issue (or service owner) has been contacted for manual testing (or will be contacted when released in alpha)
  - [ ] No testing done/necessary
- Automated tests
  - [x] Unit test(s) have been added/updated
  - [ ] Cypress E2E test(s) have been added/updated
  - [ ] No automatic tests are needed here (no functional changes/additions)
  - [ ] I want someone to help me make some tests
- UU/WCAG ([follow these guidelines](https://aksel.nav.no/god-praksis/artikler/utvikling?tema=universell-utforming) until we have our own)
  - [ ] I have tested with a screen reader/keyboard navigation/automated wcag validator
  - [x] No testing done/necessary (no DOM/visual changes)
  - [ ] I want someone to help me perform accessibility testing
- User documentation @ [altinn-studio-docs](https://github.com/Altinn/altinn-studio-docs)
  - [ ] Has been added/updated
  
  - [x] No functionality has been changed/added, so no documentation is needed
  - [ ] I will do that later/have created an issue
  
- Support in Altinn Studio
  - [ ] Issue(s) created for support in Studio
  
  - [x] This change/feature does not require any changes to Altinn Studio
- Sprint board
  - [x] The original issue (or this PR itself) has been added to the Team Apps project and to the current sprint board
  - [ ] I don't have permissions to do that, please help me out
- Labels
  - [x] I have added a `kind/*` label to this PR for proper release notes grouping
  - [ ] I don't have permissions to add labels, please help me out

### CODE REVIEW

File: src/language/sharedLanguage.ts [hunk 1]
```diff
@@ -72,7 +72,11 @@ const parserOptions: HTMLReactParserOptions = {
      * Replace p tag with Paragraph component from design system
      */
     if (isElement(domNode) && domNode.name === 'p') {
-      return React.createElement(Paragraph, { size: 'sm' }, domToReact(domNode.children as DOMNode[], parserOptions));
+      return React.createElement(
```
Comment (cammiida) [inline src/language/sharedLanguage.ts:75]: Hvorfor bruker man `React.createElement` i stedet for TSX her egentlig? 😄

**Review by cammiida — APPROVED** [review]
LGTM 💯

File: src/language/sharedLanguage.ts (hunk 1 above)
Comment (bjosttveit) [reply src/language/sharedLanguage.ts:75]: Tja, ikke noen spesiell grunn egentlig, fila var opprinnelig .ts i guess 🤷‍♂️

Comment (bjosttveit) [reply src/language/sharedLanguage.ts:75]: Jeg merger nå, men det er bare å endre det hvis du vil rydde opp 🙌

**Pull request merged**